skimage/measure/fit.py

```diff
@@ -428,6 +428,20 @@ def estimate(self, data):
         float_type = np.promote_types(data.dtype, np.float32)
         data = data.astype(float_type, copy=False)
 
+        # normalize value range to avoid misfitting due to numeric errors if
+        # the relative distanceses are small compared to absolute distances
+        origin = data.mean(axis=0)
+        data = data - origin
+        scale = data.std()
```

Comment:
Good question... Frankly speaking, I did not think too much about it and just felt more comfortable with statistical quantities. However, after thinking about it, I still would prefer the standard deviation, because the max would be sensitive to strong outliers which in a fitting scenario may appear. Therefore, I would prefer the points to scatter around an ellipse with semi-major/minor axis about one instead of forcing all values to be less or equal one.

Reply:
Note further, that `std` in this case is the same as the root mean square, because the mean value after shifting is zero. Even though this is not the bottleneck, we could calculate the scale using `np.linalg.norm(data)/np.sqrt(data.size)`.

Reply:
That reasoning sounds good to me. I'd probably leave it at `.std()` for now for readabilities sake.